bitbots_utils/launch/audio.launch

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="sim" default="false" description="If true, we don't capture audio" />
+    <arg name="dst" default="appsink"/>
+    <arg name="device" default=""/>
+    <arg name="format" default="wave"/>
+    <arg name="bitrate" default="128"/>
```

Comment:
Just out of interest on what basis have you decided for the sampling/depth/bitrate options?

Reply:
Most of the defaults were taken from [here](https://github.com/bit-bots/audio_common/blob/6f8b59377f0bbaa78697414627d6ae07ef110aad/audio_capture/launch/capture.launch.xml).

Now that I think about it, we could just include that file instead of having the audio.launch file here.